cmd/goal/ledger.go

```diff
@@ -17,13 +17,30 @@
 package main
 
 import (
+	"bytes"
 	"fmt"
+	"strconv"
 
 	"github.com/spf13/cobra"
+
+	"github.com/algorand/go-algorand/protocol/transcode"
+)
+
+var (
+	rawFilename    string
+	rawBlock       bool
+	base32Encoding bool
+	strictJSON     bool
 )
 
 func init() {
 	ledgerCmd.AddCommand(supplyCmd)
+	ledgerCmd.AddCommand(blockCmd)
+
+	blockCmd.Flags().StringVarP(&rawFilename, "out", "o", stdoutFilenameValue, "The filename to dump the raw block to (if not set, use stdout)")
```

Comment:
nit: drop the `raw` from the description here.

Reply:
done!